ApplicationLibCode/ProjectDataModel/Rim3dView.cpp

```diff
@@ -153,6 +153,14 @@ Rim3dView::Rim3dView()
 
     CAF_PDM_InitFieldNoDefault( &m_fontSize, "FontSize", "Font Size" );
 
+    CAF_PDM_InitFieldNoDefault( &m_annotationStrategy, "AnnotationStrategy", "Annotation Strategy" );
+    CAF_PDM_InitField( &m_annotationCountHint, "AnnotationCountHint", 5, "Annotation Count Hint" );
+    CAF_PDM_InitField( &m_useCustomAnnotationStrategy,
+                       "UseCustomAnnotationStrategy",
+                       false,
+                       "Use Custom Annotation Strategy",
+                       "Specify the strategy to be applied on all screen space annotations." );
```

Comment:
Make scriptable?

Reply:
I think we should use it a bit more, and maybe adjust the name in GUI and name of enums before we make it scriptable.